app/views/reviews/form/_more_about_your_stay.html.erb

```diff
@@ -0,0 +1,64 @@
+<h2><%= t('reviews.form.lastly_about_your_visit:') %></h2>
+
+<%= render partial: 'shared/radio_slider', locals: {
+  form: form,
+  field: :how_much,
+  legend: t('reviews.form.how_much_of_the_space_did_you_use?'),
+  values: Review.how_muches.keys.reverse,
+  labels: Review.how_muches.keys.reverse.map { |value|
+    render partial: 'reviews/form/radio_text_label', locals: { text: t("reviews.data.#{value}") }
+  },
+  custom_input: capture do %>
+    <div data-show-on-radio-button-value-target="elementToShow">
+      <%= form.label :how_much_custom, t('reviews.data.custom_how_much') %>
+      <%= form.text_field :how_much_custom %>
+    </div>
+  <% end
+} %>
```

Comment:
Maybe worth making these radio_sliders into helpers for form so you could do `form.radio_slider`?

Reply:
Ooh! 

Sounds like a great way to do it! How would I do that for simple_form, if we are migrating to simple_form?

Should it be part of this PR, or a new one?

Reply:
We can do it in a seperate PR  when we migrate to simple_form there is [documentation](https://github.com/heartcombo/simple_form#custom-inputs) for it